### ISSUE TITLE
[Nuxt 3.7.2] Error: Cannot read properties of undefined (reading 'use')

### ISSUE THREAD

**Issue by nathanchase** [nuxt/nuxt#23166]
### Environment

System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor
    Memory: 20.00 GB / 23.44 GB
    Container: Yes
    Shell: 5.8.1 - /usr/bin/zsh
  Binaries:
    Node: 18.15.0 - ~/.nvm/versions/node/v18.15.0/bin/node
  Browsers:
    Chrome: 111.0.5563.64

### Reproduction

https://stackblitz.com/edit/github-xntdt3?file=package.json

### Describe the bug

When running [pinia-plugin-persistedstate](https://github.com/prazdevs/pinia-plugin-persistedstate) with Nuxt 3.7.2, an error occurs:
```
[nuxt] [request error] [unhandled] [500] Cannot read properties of undefined (reading 'use')
  at eval (./node_modules/@pinia-plugin-persistedstate/nuxt/dist/runtime/plugin.mjs:10:9)  
  at eval (./virtual:nuxt:/home/projects/github-xntdt3/.nuxt/plugins/server.mjs:36:89)  
  at Module.executeAsync (./node_modules/unctx/dist/index.mjs:130:19)  
  at setup (./virtual:nuxt:/home/projects/github-xntdt3/.nuxt/plugins/server.mjs:36:65)  
  at eval (./node_modules/nuxt/dist/app/nuxt.js:114:60)  
  at fn (./node_modules/nuxt/dist/app/nuxt.js:162:44)  
  at Object.callAsync (./node_modules/unctx/dist/index.mjs:87:55)  
  at eval (./node_modules/nuxt/dist/app/nuxt.js:164:56)  
  at Object.runWithContext (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:3855:18)  
  at callWithNuxt (./node_modules/nuxt/dist/app/nuxt.js:164:24)
  ```
Downgrading to Nuxt 3.7.1, it works: https://stackblitz.com/edit/github-xntdt3-fzhfhx?file=package.json

Crossposted: https://github.com/prazdevs/pinia-plugin-persistedstate/issues/239

**Comment by martinszeltins** [comment]
+1 upgrade to 3.7.2 caused this error for me 😞

**Comment by okwinwin** [comment]
+1 upgrade to 3.7.2 caused this error for me too 😞😠

**Comment by danielroe** [comment]
I plan to release a hotfix for this by the end of today; you can try with the [edge channel](https://nuxt.com/docs/guide/going-further/edge-channel/) in the mean time.

**Comment by nathanchase** [comment]
Can confirm this issue is resolved with Nuxt 3.7.3. Thank you, @danielroe!

**Comment by danielroe** [comment]
Great! And sorry for the pain here.
